cat/test/main.c

```diff
@@ -2422,6 +2422,28 @@ extract_reference_files(const char **names)
 		extract_reference_file(*names++);
 }
 
+
+/*
+ * Tests if the char array is an absolute path.
+ */
+int
+is_absolute_path(char *testdir)
+{
+	int is_absolute  = testdir[0] != '/';
```

Comment:
Since this value is ignored for Windows platforms (overridden immediately below), it should be in a `#else`